packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/AppContext.kt

```diff
@@ -264,8 +265,13 @@ class AppContext(
   @Suppress("UNCHECKED_CAST")
   @UiThread
   fun <T : View> findView(viewTag: Int): T? {
-    val rnUIManager = reactContextHolder.get()?.getNativeModule(UIManagerModule::class.java)
-    return rnUIManager?.resolveView(viewTag) as? T
+    return if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
```

Comment:
Let's use `RN_FABRIC_ENABLED` as it works on both platforms 😉
```suggestion
    return if (BuildConfig.RN_FABRIC_ENABLED) {
```

Reply:
This flag isn't available in the `BuildConfig` 🤷